### DISCUSSION TITLE
Check temporary_directory for *.tmp files and delete them when starting manager

### DISCUSSION THREAD

**Discussion by schokoladi** [swar/Swar-Chia-Plot-Manager#115] [Q&A]
If I change something regarding the chia plotting, I normally stop the manager, make the changes, delete temp files and then start the manager again. Unfortunately, sometimes I forget to delete *.tmp files before starting the plot manager (which is very useful btw!) which can prevent plot creation because the directory is full. Would be nice if you added this as an option or automatic check.

**Comment by swar** [comment]
This is not an issue, moving it to a discussion.

Also, if you stop manager, your plots are still running in the background so you may be deleting files that are being openly accessed. But I will say that I have no intention of deleting any .tmp file as that is the user's responsibility.
